hbase-client/src/main/java/org/apache/hadoop/hbase/client/ConnectionImplementation.java

```diff
@@ -189,9 +189,13 @@ public class ConnectionImplementation implements ClusterConnection, Closeable {
   final int rpcTimeout;
 
   /**
-   * Global nonceGenerator shared per client.Currently there's no reason to limit its scope. Once
+   * Global nonceGenerator shared per client. Currently there's no reason to limit its scope. Once
    * it's set under nonceGeneratorCreateLock, it is never unset or changed.
    */
+  // XXX: It is a bad pattern to assign a value to a static field from a constructor. However
```

Comment:
Declined to fix the issue because we'd have to use a Configuration instance at static class init time, not the Configuration that gets passed into the Constructor. However, marked it as a code smell with `XXX`.